src/dotnet/Chat.UI.Blazor/Components/components.css

```diff
@@ -58,7 +58,7 @@
     @apply animate-display-from-bottom-slow;
 }
 .recording-elsewhere-sub-header-content {
-    @apply flex-1 flex-x items-center justify-center gap-x-2 px-4 w-full;
+    @apply flex-1 flex-x items-center justify-center gap-x-2 px-4 w-full flex-wrap md:flex-nowrap;
```

Comment:
Suspicious long style line - probably make sense to wrap.

Reply:
done